### PR TITLE
Release Notes for Google 3.0.2

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates the Google Shopping ads Channel Release Notes for version 3.0.2, the Release page, and the compatibility.yml file.

## Affected DevDocs pages

-  https://devdocs.magento.com/extensions/google-shopping-ads/release-notes/
- https://devdocs.magento.com/release/

whatsnew
Release Notes have been updated for the Google Shopping ads Channel v3.0.2 release.

### CODE REVIEW

File: extensions/google-shopping-ads/release-notes/index.md [hunk 1]
```diff
@@ -23,6 +23,18 @@ The release notes include:
 -  {:.fix}Fixes and improvements
 -  {:.bug}Known issues
 
+### v3.0.2
+
+Google Shopping ads Channel 3.0.2 is compatible with versions 2.2.4+, 2.3.0, 2.3.1, and 2.3.2 of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+
+-  {:.new}**Admin Panel Access**: <!--CHAN-3815-->Improved the onboarding process to allow access to the side _Admin_ panel when the initial _Welcome to Google Shopping ads Channel in Magento_ screen displays.
+
+-  {:.fix}**Shopping Cart**: <!--CHAN-3654-->Corrected an issue causing an error when attempting to delete an item from the shopping cart.
```
Comment (shrielenee) [inline extensions/google-shopping-ads/release-notes/index.md]:
```suggestion
-  {:.fix}**Shopping Cart**: <!--CHAN-3654-->Corrected an issue that caused an error when attempting to delete an item from the shopping cart.
```

File: extensions/google-shopping-ads/release-notes/index.md [hunk 2]
```diff
@@ -23,6 +23,18 @@ The release notes include:
...
+
+-  {:.fix}**Add to Cart**: <!--CHAN-3861-->Corrected an issue causing an error when hovering over an item on the store landing page and clicking **Add to Cart**. With this release, if all item required item options (size, color, etc) are not complete when attempting to **Add to Cart**, a message appears stating that you must choose options for your item.
```
Comment (shrielenee) [inline extensions/google-shopping-ads/release-notes/index.md]:
```suggestion
-  {:.fix}**Add to Cart**: <!--CHAN-3861-->Corrected an issue that caused an error when hovering over an item on the store landing page and clicking **Add to Cart**. With this release, if all item required item options (size, color, etc.) are not complete when attempting to **Add to Cart**, a message appears stating that you must choose options for your item.
```

File: release/index.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ The following table describes the status of Magento software availability and wh
...
 | **Amazon Sales Channel 3.0.1**                    | Available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x (US, Canada, Mexico, UK)                                                                                                                                                                                                                    | [Magento Marketplace](https://marketplace.magento.com/magento-module-amazon.html)                 |
-| **Google Shopping ads Channel 3.0.1**             | Available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x <br>Bundled in Magento starting with 2.3.3, but still available on the Marketplace | [Magento Marketplace](http://marketplace.magento.com/magento-google-shopping-ads.html)            |
+| **Google Shopping ads Channel 3.0.2**             | Available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x <br>Bundled in Magento starting with 2.3.3, but still available on the Marketplace | [Magento Marketplace](http://marketplace.magento.com/magento-google-shopping-ads.html)            |
```
Comment (shrielenee) [inline release/index.md]:
```suggestion
| **Google Shopping ads Channel 3.0.2**             | Google Shopping ads Channel 3.0.2 is available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x. <br>It is bundled in Magento starting with 2.3.3, but still available on the Marketplace. | [Magento Marketplace](http://marketplace.magento.com/magento-google-shopping-ads.html)            |
```

Comment (hickskenh) [reply release/index.md]: to keep with the formatting of above line items, declining this suggestion

**Comment by hickskenh** [comment]
running tests

**Pull request merged**